make/common/Modules.gmk

```diff
@@ -231,6 +231,16 @@ ifeq ($(call isTargetOs, windows macosx linux), false)
   MODULES_FILTER += jdk.incubator.jpackage
 endif
 
+################################################################################
+# Filter out specific modules for mobile platforms
```

Comment:
Is there a reason why you want to excluded these modules?  I would think that it might be nice to be able to manage a mobile application remotely.   If these modules are incompatible with native-image, then just remove them from your built images when you create a GraalVM builder image.

Reply:
They are included again now.

Reply:
But jdk.sctp is still left out on ios?

Reply:
Why don't just exclude GLX (Desktop OpenGL), or port to EGL with OpenGL ES because both Android and iOS does not have Desktop OpenGL (althought GL4ES is a solution).

Reply:
Excluding GLX would be good. I'm looking into that, but there are other files that need to be excluded as well (java2d/x11).
The port to EGL/OpenGL ES is exactly what is done in the OpenJFX project, and therefore it seems a bit redundant to me to re-introduce that here.

Reply:
correct, that one is still failing on ios (Sctp.h includes linux/types.h)

Reply:
@johanvos If you look at make/modules/jdk.sctp/Lib.gmk, you see that the native library is excluded on macosx and aix.

I'm not sure about the details here, maybe sctp support is not possible on iOS, but maybe the macos pure java implementation is what's needed on iOS as well?

Reply:
Heh. I just had a look at the macos implementation, or should I say "implementation". It's basically:

```
    private static final String message = "SCTP not supported on this platform";

    public SctpChannelImpl(SelectorProvider provider) {
        super(provider);
        throw new UnsupportedOperationException(message);
    }
```

so I dunno, maybe it's just as well to skip the entire module.